src/io/procops.c

```diff
@@ -49,29 +49,45 @@ static char * ANSIToUTF8(MVMuint16 acp, const char * str)
     return result;
 }
 
+MVM_PUBLIC char **
```

Comment:
It's needed because it's being called from `main.c`, which is an executable that dynamically links against `moar.dll`. We can mark it `MVM_PUBLIC`, but in that case must also follow the naming convention of having an `MVM_` prefix, so we don't cause namespace pollution and make problems for anyone embedding MoarVM. Also I suggest it also be added to `procops.h`, which will eliminate the need for the decl in `main.c`.